examples/headless-example/src/components/App.svelte

```diff
@@ -153,28 +164,35 @@
 
   let nextWorkflow;
   let shouldResubmit = false;
+  $: isCompleted = $workflowQuery.data?.workflowRuntimeData?.status === 'completed';
```

Comment:
we have a lot of null coalescing operators among the project; would it make sense to enforce typings so we would have only the required ones?

Reply:
In this case it makes sense since data is undefined at times (@tanstack/query behaviour). Some other cases are due to Prisma where we would need to discuss. Otherwise I agree with you.